packages/shared/src/hooks/utils/useThemedAsset.ts

```diff
@@ -19,6 +20,7 @@ export const useThemedAsset = (): UseCloudinaryAsset => {
   }, [themeMode]);
 
   return {
+    isLight,
```

Comment:
Hmm I get why you need it but not sure `isLight` is something that should be returned from this hook since I would expect all keys are specific asset keys. It would more fit useColorMode or useTheme that would just return `light | dark`.

Reply:
Agree with that. Would be better off in a separate hook or rename this hook completely. Though for the moment, we will close this PR as we have decided that the fix should actually be from the asset to be having the same ratio and size since the variation is technically of the "same" image.

Reply:
It seems we will have to keep this PR, but instead update the links and the usage of the assets 😅